tests/fute/Makefile

```diff
@@ -0,0 +1,33 @@
+PYTHON_CMD:=source venv/bin/activate && source .secrets && python3
+
+all: venv check
+
+venv:
+	python3 -m venv venv
+	source venv/bin/activate && pip3 install https://github.com/CSCfi/tauhka/archive/master.zip
```

Comment:
```
/bin/sh: 1: source: not found
```
Annoyingly, depending on the system, /bin/sh may or may not be a symlink to bash which supports source.

Maybe use . which is more portable.

Reply:
there is a bug, missing SHELL:=/bin/bash

Reply:
this is now fixed